tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -1011,9 +1001,9 @@
     "value": {
         "params": {
             "column_list": {"schema": {"type": "array"}, "value": ["a", "b", "c"]},
-            "v__0": {"schema": {"type": "string"}, "value": "a"},
-            "v__1": {"schema": {"type": "string"}, "value": "b"},
-            "v__2": {"schema": {"type": "string"}, "value": "c"},
+            "column_list_0": {"schema": {"type": "string"}, "value": "a"},
+            "column_list_1": {"schema": {"type": "string"}, "value": "b"},
+            "column_list_2": {"schema": {"type": "string"}, "value": "c"},
```

Comment:
Another bug. `$column_list_0` would not have been substituted by the `v__0` param.